app/workers/scm/create_repository_job.rb

```diff
@@ -43,7 +43,8 @@ def initialize(repository)
 
   def perform
     # Create the repository locally.
-    mode = config[:mode] || default_mode
+    # chmod requires integer value
+    mode = (config[:mode] || default_mode).to_i
```

Comment:
Holy moly, that should have been `to_i(8)`, but even then, this may fail when already providing an integer.

I'll have to rework this and test it more extensively.